cf/test/test_Data.py

```diff
@@ -492,9 +492,6 @@ def test_Data_halo(self):
             d.halo(4)
 
     def test_Data_mask(self):
-        if self.test_only and inspect.stack()[0][3] not in self.test_only:
-            return
-
```

Comment:
This is an appropriate time to consider my `TODODASK` note just under this line, namely that it may make sense, given the similarities in the methods, to combine `test_Data_apply_masking` and `test_Data_mask`. What do you think? I'm happy to either combine or not, so please either go ahead and do that, else just remove the `TODODASK` comment lines just below. Thanks.

Reply:
Perhaps go with the easy option of not combining at this time - so I'll just remove the message!